### PR TITLE
Provide more appropriate naming for the fourth and fifth arguments of BN

### PR DESCRIPTION
**Pull request by ir5**
Ad-hoc approach

### CODE REVIEW

File: onnx_chainer/context.py [hunk 1]
```diff
@@ -54,3 +55,24 @@ def is_pinned(self, variable):
         if str_id not in self.name_list:
             return False
         return self.name_list[str_id][1]
+
+    def add_param(self, array, name, use_original_name=False):
+        """Add array to context parameter
+
+        To be converted as ONNX tensor.
+
+        Return:
```
Comment (shinh) [inline onnx_chainer/context.py]: Returns

File: onnx_chainer/functions/normalization.py [hunk 2]
```diff
@@ -10,34 +10,50 @@
 @support((1, 6, 7))
 def convert_BatchNormalization(func, opset_version, input_names,
                                output_names, context, parameters):
+    names = [context.get_name(v.get_variable().array) for v in func.inputs]
+    if names[1].startswith('param_'):
+        prefix = names[1][:-6]  # remove "_gamma"
```
Comment (shinh) [inline onnx_chainer/functions/normalization.py]: `[:-1-len('_gamma')]`?

File: onnx_chainer/functions/normalization.py [hunk 3]
```diff
@@ -10,34 +10,50 @@
...
+    elif names[2].startswith('param_'):
+        prefix = names[2][:-5]  # remove "_beta"
```
Comment (shinh) [inline onnx_chainer/functions/normalization.py]: ditto?

File: onnx_chainer/context.py (hunk 1 above)
Comment (ir5) [reply onnx_chainer/context.py]: modified.

File: onnx_chainer/functions/normalization.py (hunk 2 above)
Comment (ir5) [reply onnx_chainer/functions/normalization.py]: It seems that this part will be entirely changed in #185. I will keep this part as is in this PR.

**Pull request merged**